jenkins_pipelines/environments/uyuni-prs-ci-tests

```diff
@@ -19,7 +19,7 @@ node('pull-request-test') {
             string(name: 'cucumber_gitrepo', defaultValue: 'https://github.com/uyuni-project/uyuni.git', description: 'Change this by your repo, only if you changed the tests in your PR'),
             string(name: 'cucumber_ref', defaultValue: 'master', description: 'Change this by your branch, only if you changed the tests in your PR'),
             string(name: 'email_to', defaultValue: '', description: 'Receive a report to your e-mail when it finish'),
-            string(name: 'build_repo', defaultValue: 'openSUSE_Leap_15.2', description: 'Repository to build the packages for'),
+            string(name: 'build_repo', defaultValue: 'openSUSE_Leap_15.3', description: 'Repository to build the packages for'),
```

Comment:
I am confused, why don't you use this value for the URLs? That would make maintenance easier.

Reply:
that is a good idea. Let me change it and push it again.

Reply:
done